### DISCUSSION TITLE
SvelteKit app failed to load resources. (blocked by CORS policy)

### DISCUSSION THREAD

**Discussion by ptsgrn** [vercel/community#1964] [Help] [answered]
### Summary

I am encountering an issue while deploying my personal website, created using SvelteKit, to Vercel. While the website runs without any errors on my local setup, on visiting https://patsagone.vercel.app/, I noticed that none of the scripts or stylesheets are loading. On checking the browser console, the following error message is displayed:
"Access to script at 'https://vercel.com/deployments/patsagone.vercel.app/source' (redirected from 'https://patsagone.vercel.app/_src/immutable/chunks/index-xxxxxxx.js') from origin 'https://patsagone.vercel.app/' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource."

### Additional information

I have updated my SvelteKit app to version 1.x and have successfully run it on both dev and build preview locally. However, the deployment on Vercel resulted in plain HTML without any errors reported during the build process.

I have previously deployed my SvelteKit app to Vercel and it had been working without any issues. However, I encountered this error only in the last deployment after making some updates to the app.

### Example

https://patsagone.vercel.app/

**Comment by amyegan** [comment]
Hi @ptsgrn! I don't see any error codes in my network tab when I visit https://patsagone.vercel.app/

Did you already find a solution?

**Comment by ptsgrn** [comment, marked as answer]
Hey @amyegan, I actually figured out the problem a couple of hours ago 😅. So, it turns out that I accidentally added an `appDir` configuration in svelte.config.js pointing to `_src`. But Vercel has a security policy that restricts access to `_src` and `_logs`, so putting production code there wasn't a good decision as it requires login and may not serve any assets. Thanks for taking the time to look into my case, I really appreciate it!
